src/ui/layout/CreateButton.vue

```diff
@@ -56,6 +42,26 @@ export default {
         }
     },
     methods: {
+        getItems() {
```

Comment:
Does getItems need to be a class method? If nothing else is using it, can't this be done in the computed section itself?

Reply:
Ok I'll take a look!

Reply:
So this is only called by sortedItems, which is a computed value already, do you think it would make sense to also make this a computed? Or move this code into the sortedItems computed variable?

Reply:
LEt's leave it as is for now.